module-services/service-fileindexer/StartupIndexer.cpp

```diff
@@ -12,29 +12,66 @@
 
 #include <filesystem>
 #include <fstream>
+#include <optional>
 
 namespace service::detail
 {
     namespace fs = std::filesystem;
-    namespace
+    using namespace std::literals;
+    using namespace std::chrono_literals;
+
+    const auto lock_file_name        = purefs::dir::getUserDiskPath() / ".directory_is_indexed";
```

Comment:
Same here, consider using function

Reply:
I believe, in this case, it's not applicable. This path is used exclusively by this module to perform its logic.